src/common/mathjax.js

```diff
@@ -53,7 +53,7 @@ goog.require('sre.System');
 
   (sre.System.getInstance()).setupEngine(
       {mode: sre.Engine.Mode.HTTP,
-       json: MathJax.Ajax.config.path['SRE'],
+       json: MathJax.Ajax.config.path['SRE'] + '/mathmaps/',
```

Comment:
I think this should be 

`json: MathJax.Ajax.config.path['SRE'] + 'mathmaps/',`

Without the leading `/` for `mathmaps`.